specification/network/resource-manager/Microsoft.Network/stable/2020-08-01/networkManagerGroup.json

```diff
@@ -0,0 +1,505 @@
+          },
+          {
+            "$ref": "#/parameters/NetworkGroupNameParameter"
+          },
+          {
+            "$ref": "./network.json#/parameters/ApiVersionParameter"
+          },
+          {
+            "$ref": "#/parameters/ListPageSizeParameter"
+          },
+          {
+            "$ref": "#/parameters/ListSkipTokenParameter"
+          }
+        ],
+        "responses": {
+          "200": {
+            "description": "OK - Returns information about the network group.",
+            "schema": {
+              "$ref": "#/definitions/EffectiveVirtualNetworksListResult"
+            }
+          },
+          "default": {
+            "description": "Resource Provider error response describing why the operation failed.",
+            "schema": {
+              "$ref": "./network.json#/definitions/CloudError"
+            }
+          }
+        },
+        "x-ms-examples": {
+          "NetworkGroupsList": {
+            "$ref": "./examples/NetworkManagerEffectiveVirtualNetworksList.json"
+          }
+        }
+      }
+    }
+  },
+  "definitions": {
+    "NetworkGroup": {
+      "properties": {
+        "properties": {
+          "x-ms-client-flatten": true,
+          "$ref": "#/definitions/NetworkGroupProperties",
+          "description": "The Network Group properties"
+        },
+        "tags": {
+          "type": "object",
+          "additionalProperties": {
+            "type": "string"
+          },
+          "description": "Resource tags."
+        }
+      },
+      "allOf": [
+        {
+          "$ref": "./network.json#/definitions/ProxyResource"
+        }
+      ],
+      "description": "The Managed Network resource"
+    },
+    "NetworkGroupListResult": {
+      "properties": {
+        "value": {
+          "type": "array",
+          "items": {
+            "$ref": "#/definitions/NetworkGroup"
+          },
+          "description": "Gets a page of NetworkGroup"
+        },
+        "nextLink": {
+          "type": "string",
+          "description": "Gets the URL to get the next set of results."
+        }
+      },
+      "description": "Result of the request to list NetworkGroup. It contains a list of groups and a URL link to get the next set of results."
+    },
+    "NetworkGroupProperties": {
+      "type": "object",
+      "properties": {
+        "displayName": {
+          "type": "string",
+          "description": "A friendly name for the network group."
+        },
+        "description": {
+          "type": "string",
+          "description": "A description of the network group."
+        },
+        "memberType": {
+          "type": "string",
+          "description": "Group member Type.",
+          "enum": [
+            "VirtualNetwork",
+            "Subnet"
+          ],
+          "x-ms-enum": {
+            "name": "memberType",
+            "modelAsString": true
+          }
+        },
+        "groupMembers": {
+          "type": "array",
```

Comment:
The groupMemebersItem object has only a single property by name 'resourceId'. Any reason why groupMembers is not defined just as an array of strings? Just an array of strings simplifies the API structure.

Reply:
We had other properties before, but by latest design, all other properties has been removed.  I will discuss with my teammate, and see if we will add new properties in the future.

Reply:
After disucssing with my teammates, we need to keep the obj to add new properties in the future